### PR TITLE
chore(e2e): Run e2e against latest released serving version v0.10.0

### PR DESCRIPTION
**Pull request by navidshaikh**
## Proposed Changes
 - This changeset updates serving version for pull-knative-client-integration-tests-latest-release
 - Serving nightly release is tested in prow job pull-knative-client-integration-tests

### CODE REVIEW

**Comment by navidshaikh** [comment]
added debug code
/hold

File: test/e2e/traffic_split_test.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func formatActualTargets(t *testing.T, actualTargets []string) (formattedTargets
 
 // TestTrafficSplit runs different e2e tests for service traffic splitting and verifies the traffic targets from service status
 func TestTrafficSplit(t *testing.T) {
-	t.Parallel()
+	//t.Parallel()
```
Comment (maximilien) [inline test/e2e/traffic_split_test.go]: Is there a reason for limiting parallelism... or just temporary?

Comment (navidshaikh) [reply test/e2e/traffic_split_test.go]: just temporary

**Comment by navidshaikh** [comment]
e2e works fine against serving v0.10.0 [setup](https://prow.knative.dev/view/gcs/knative-prow/pr-logs/pull/knative_client/501/pull-knative-client-integration-tests-latest-release/1194543317308674048#1:build-log.txt%3A855), but fails against serving nightly build.
Related to #500

File: test/e2e/traffic_split_test.go (hunk 1 above)
Comment (navidshaikh) [reply test/e2e/traffic_split_test.go]: removed the commit

**Comment by navidshaikh** [comment]
/retest

The work around PR https://github.com/knative/client/pull/508 got in after this CI tests triggered.

**Pull request merged**